oauth_dropins/handlers.py

```diff
@@ -209,6 +209,11 @@ def finish(self, auth_entity, state=None):
                      ('access_token_secret', token[1])]
       except NotImplementedError:
         logging.info('access_token() not implemented')
+      try:
+        token = auth_entity.refresh_token
+        params.append(('refresh_token', token))
```

Comment:
out of curiosity, why add this? afaik we don't actually use the `access_token_*` params that we insert above either; they're only used in the template in an unused HTML comment. consider deleting?

Reply:
i might be totally misunderstanding how those are used, but is that not where the params that are asked for by granary here https://github.com/snarfed/granary/blob/6e97787c72a139709e011587a65722ec64212fe4/api.py#L116 come from?  This could explain why I'm not able to get refresh_token passed over there.

Reply:
ah, no, granary's params come from its own requests directly. the oauth dance is only used in the interactive web UI, and even then just ends up populating the granary HTML template at the end, eg:

https://github.com/snarfed/granary/blob/6e97787c72a139709e011587a65722ec64212fe4/granary/templates/index.html#L246-L258

Reply:
This was very helpful, I now get why this is here and it allowed me to modify index.html in granary to allow the demo to work!